src/serac/infrastructure/input.cpp

```diff
@@ -168,33 +168,33 @@ std::unique_ptr<mfem::Coefficient> CoefficientInputOptions::constructScalar() co
   }
 }
 
-void CoefficientInputOptions::defineInputFileSchema(axom::inlet::Table& table)
+void CoefficientInputOptions::defineInputFileSchema(axom::inlet::Container& container)
```

Comment:
do we want to be consistent on variable names in these functions between container/base?